requirements.txt

```diff
@@ -1,4 +1,4 @@
 asgiref==3.4.1
-Django==3.2
+Django
```

Comment:
My proposal is to fix major version (`Django=="4.*"`). What do you think?

Reply:
Done.